pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "member-database"
-version = "0.3.9"
+version = "0.4.0"
```

Comment:
I think 0.4.0 is fine, given the many changes we had so far. But in principle 0.3.10 would be more appropriate, no?

Reply:
We said we bump the middle version number when there are schema changes.

Reply:
Ok, then forget what I said 🙈